contribs/gmf/src/query/gridComponent.html

```diff
@@ -43,83 +40,78 @@
       id="{{ctrl.escapeValue(gridSource.source.label)}}">
 
       <ngeo-grid
-        ngeo-grid-configuration="gridSource.configuration"
-        ng-if="gridSource.source.tooManyResults !== true">
+        ngeo-grid-configuration="gridSource.configuration">
       </ngeo-grid>
-
-      <div ng-if="gridSource.source.tooManyResults === true">
-        <div class="gmf-displayquerygrid-message alert alert-warning">
-          <p><span translate>The results can not be displayed because the maximum number has been reached</span> ({{gridSource.source.limit}}).</p>
-          <p translate>Please refine your query.</p>
-        </div>
-      </div>
-    </div>
   </div>
+  <div class="container-fluid">
+  <div ng-show="!ctrl.pending && ctrl.getActiveGridSource() && ctrl.getActiveGridSource().configuration !== null" class="row">
+    <div class="col-md-5 my-auto" translate>
+      <span ng-if="ctrl.hasActiveSourceTooManyResults()" class="gmf-query-grid-too-many text-warning">Only {{ctrl.getActiveGridSourceLimit()}} of {{ctrl.getActiveGridSourceTotalFeatures()}} results displayed, as the maximum number is reached. Please refine your query. </span>
```

Comment:
I think the translation will not work like that (with the translate in the parent markup).
You should have to do this in you case I think:
`{{'Only' | translate}} {{ctrl.getActiveGridSourceLimit()}} {{'of' | translate}} {{ctrl.getActiveGridSourceTotalFeatures()}} {{'results displayed, as the maximum number is reached. Please refine your query.' | translate}}`